packages/bot-web-ui/src/components/run-panel/run-panel.tsx

```diff
@@ -9,10 +8,55 @@ import Transactions from 'Components/transactions';
 import TradeAnimation from 'Components/trade-animation';
 import SelfExclusion from 'Components/self-exclusion';
 import { popover_zindex } from 'Constants/z-indexes';
-import { connect } from 'Stores/connect';
 import { isMobile } from '@deriv/shared';
+import { useDBotStore } from 'Stores/useDBotStore';
+import { useStore } from '@deriv/stores';
+import { observer } from 'mobx-react';
```

Comment:
observer should come from '@deriv/stores'

Reply:
done ✅